src/block/defines.rs

```diff
@@ -9,6 +9,8 @@ pub const VIRTIO_BLK_T_IN: u32 = 0;
 pub const VIRTIO_BLK_T_OUT: u32 = 1;
 /// Flush request.
 pub const VIRTIO_BLK_T_FLUSH: u32 = 4;
+/// Get device ID request.
+pub const VIRTIO_BLK_T_GET_ID: u32 = 8;
```

Comment:
Hmm, just wondering, where do the added values come from? (it looks like they're not in the 1.1 standard). Are they about to be to added to a subsequent version? We could also say a few words about this to improve clarity.